docs/workflow.md

```diff
@@ -67,11 +98,13 @@ stateDiagram-v2
         PreventIndexation --> Secure
         Secure --> Adminer
         Adminer --> Couscous
+        Couscous --> Remove files
```

Comment:
Spaces aren't supported here, see https://mermaid.js.org/syntax/stateDiagram.html#spaces-in-state-names.

```suggestion
        Couscous --> RemoveFiles
        state "Remove files" as RemoveFiles
```